tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -86,3 +98,154 @@ Verify Service Provides "Get Started" Button In The Explore Page
   Wait Until Page Contains Element    ${ODH_DASHBOARD_SIDEBAR_HEADER_TITLE}   timeout=10   error=${app_name} does not have sidebar with information in the Explore page of ODS Dashboard
   Page Should Contain Element    ${ODH_DASHBOARD_SIDEBAR_HEADER_GET_STARTED_ELEMENT}   message=${app_name} does not have a "Get started" button in ODS Dashboard
 
+Check HTTP Status Code
+    [Arguments]  ${link_to_check}  ${expected}=200
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    Run Keyword And Continue On Failure  Status Should Be  ${expected}
+    [Return]  ${response.status_code}
+
+Load App Info JSON File
+    ${APPS_DICT}=  Load Json File  ${APPS_DICT_PATH}
+    ${APPS_DICT}=  Set Variable  ${APPS_DICT}[apps]
+    Set Suite Variable  ${APPS_DICT}
+
+Wait Until Cards Are Loaded
+    Wait Until Page Contains Element    xpath://div[contains(@class,'odh-explore-apps__gallery')]
+
+Get App ID From Card
+    [Arguments]  ${card_locator}
+    ${id}=  Get Element Attribute    xpath:${card_locator}    id
+    [Return]  ${id}
+
+Check Number of Cards
+    ${N_TILES}=  Get Element Count    xpath:${TILES_XP}
+    Set Suite Variable  ${N_TILES}
```

Comment:
Should be returned instead of set

Reply:
done in this way to make the TC more abstract without passing arguments